clcache.py

```diff
@@ -1612,19 +1618,28 @@ def processCompileRequest(cache, compiler, args):
     printOutAndErr(out, err)
     return exitCode
 
+def formBaseCommandLine(cmdLine, sourceFiles):
+    # type: (List[str], List[Tuple[str, str]]) -> List[str]
+    setOfSources = set([sourceFile for sourceFile, _ in sourceFiles])
+    skippedArgs = ('/MP', '/Tc', '/Tp')
```

Comment:
I _think_ that `-Tc` and `-Tp` work, too. At least we support either form in `CommandLineAnalyzer.analyze`, and with Visual Studio 2013 (the only compiler I have at hand right now) you can indeed use either form.